jekyll/_cci2/env-vars.md

```diff
@@ -317,7 +317,9 @@ Once created, environment variables are hidden and uneditable in the application
 
 ## Setting an Environment Variable in a Container
 
-Environment variables can also be set for a Docker container. To do this, use the [`environment` key]({{ site.baseurl }}/2.0/configuration-reference/#docker--machine--macos--windows-executor). **Note**: Environment variables set in this way are not available to _steps_ run within the container. These environment variables are only available to the entrypoint/command run by the container. See the [Custom Images]({{ site.baseurl }}/2.0/custom-images/#adding-an-entrypoint) guide for further information about using a container's entrypoint.
+Environment variables can also be set for a Docker container. To do this, use the [`environment` key]({{ site.baseurl }}/2.0/configuration-reference/#docker--machine--macos--windows-executor). 
+
+**Note**: Environment variables set in this way are not available to _steps_ run within the container, they are only available to the entrypoint/command run _by_ the container. By default, CircleCI will ignore the entrypoint for a job's primary container. If you wish to use container environment variables, you will need to preserve the entrypoint. For more information, see the _adding an entrypoint_ section of the [Custom Images]({{ site.baseurl }}/2.0/custom-images/#adding-an-entrypoint) guide.
```

Comment:
Also, only because I'm a stickler for hyperlinks, personally I'd expand the link:

```suggestion
**Note**: Environment variables set in this way are not available to _steps_ run within the container, they are only available to the entrypoint/command run _by_ the container. By default, CircleCI will ignore the entrypoint for a job's primary container. If you wish to use container environment variables, you will need to preserve the entrypoint. For more information, see [the _adding an entrypoint_ section of the Custom Images guide]({{ site.baseurl }}/2.0/custom-images/#adding-an-entrypoint).
```